### DISCUSSION TITLE
Error! Build failed

### DISCUSSION THREAD

**Discussion by lonelydatum** [vercel/next.js#12428] [Help]
Hello,

I have a very simple project with 2 files and I can't get it to deploy. It works fine on localhost.

Here is the repo:
https://github.com/lonelydatum/support_help_vercel

The error message from the CLI  is not helpful
Error! Build failed
Error! Check your logs
![image](https://user-images.githubusercontent.com/858774/80897606-760af980-8cc8-11ea-9383-8ce21b07d77b.png)

It seemed to work when I removed the engines version yesterday from my package.json but it has now mysteriously broke again. I'm not even sure if it is related to the node version at this point. 

Below are my specs:

package.json
` "engines": {
    "node": "12.x"
  }`

node -v
v12.4.0

now -v 
Now CLI 18.0.0

Thank you.

**Comment by Timer** [comment]
The application deployed fine for me:
https://support-help-vercel-fyi5h0lcr.now.sh/

Why are you trying to set `NODE_PATH`? You might have to delete this.

**Reply by lonelydatum** [reply]
Thanks for the reply.

This is so strange. It's deploying, though I didn't make any changes. 

Over the last couple of days, the build would just fail and then for no reason would start working again. Is this a glitch on your end? 

What makes this so difficult to fix is that the error just says Error! Build failed Error! Check your logs.
